aiida/orm/implementation/django/computer.py

```diff
@@ -19,7 +19,7 @@
 from aiida.orm.implementation.general.computer import AbstractComputer, Util as ComputerUtil
 from aiida.common.exceptions import (NotExistent, ConfigurationError,
                                      InvalidOperation, DbContentError)
-from aiida.orm.implementation.django.utils import get_db_columns
+from aiida.orm.implementation.sqlalchemy.utils import get_db_columns
```

Comment:
Why the django implemenation depends on the sqlalchemy one? If the function is common, move it out in the general folder, otherwise fix this